tensorflow-whl/Dockerfile

```diff
@@ -0,0 +1,105 @@
+FROM nvidia/cuda:9.2-cudnn7-devel-ubuntu16.04 AS nvidia
```

Comment:
This Dockerfile would be easier to review if it would show the diffs from the file you copied it from (and then modified).  Isn't this possible with git?  If not would it help (in the future) to first to a straight copy, then apply deltas?

Reply:
I don't think that is possible or at least I am unaware of how to do it.